client/src/ui/pages/ProfilePage.tsx

```diff
@@ -1,11 +1,55 @@
 import("../../App.css");
 import Navigation from "../components/Navigation.tsx";
+import Profile from "../components/Profile.tsx";
+import background from "../assets/images/background.png";
 
 export default function ProfilePage() {
+  // SQL functions
+  function auth() {
```

Comment:
Same stuff here like I mentioned for the list() function in Profile.tsx, I'd just move these to state variables for now with default values, easier to edit later and less messy